src/middleware/node/middleware.ts

```diff
@@ -72,7 +85,7 @@ export async function middleware(
     didTimeout = true;
     response.statusCode = 202;
     response.end("still processing\n");
-  }, 9000).unref();
```

Comment:
This is needed for Deno
https://github.com/octokit/octokit.js/issues/2079

Reply:
I read what unref is and why it is used, and I'm almost certain that we don't "need" that unref (we have a small timeout and also we clear in every paths). Also, v8 does not have unref, so cloudflare workers die BCS of it.

Probably we should use the below check instead of just adding/deleting it;
```js
if (typeof timeout.unref === "function") {
  timeout.unref();
}
```

Reply:
> Also, v8 does not have unref, so cloudflare workers die BCS of it

are you sure? Node is built on v8

Reply:
I faced the "unref is not a function" error when I tried to run it on cloudflare workers so I'm sure. But a fixup commit is on the way.

Reply:
I added this back with an if, and I left out the else from the branch coverage, to keep it 100%. I don't think it is worth the effort to mock the `setInterval` out.